state/OffenceStore/useCreateOffence.ts

```diff
@@ -32,6 +34,8 @@ export const useCreateOffence = () => {
     resolutionType,
   } = useOffenceStoreContext((state) => state)
 
+  const [isLoading, setLoading] = useState(false)
```

Comment:
As we discussed, setting the GPS could be moved to mutation together with `clientApi.scanControllerCreateOffence`